gov2/cloudfront/CreateDistribution/CreateDistribution_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+func TestCreateDistribution(t *testing.T) {
+	thisTime := time.Now()
+	nowString := thisTime.Format("2006-01-02 15:04:05 Monday")
+	t.Log("Starting integration test at " + nowString)
+
+	sdkConfig, err := config.LoadDefaultConfig(context.TODO())
+
+	if err != nil {
+		t.Log("Got an error ...:")
+		t.Log(err)
+		return
+	}
+
+	s3Client := s3.NewFromConfig(sdkConfig)
+	cloudfrontClient := cloudfront.NewFromConfig(sdkConfig)
+	bucketName := "<EXAMPLE-BUCKET-NAME>"
+	certificateSSLArn := "<AWS CERTIFICATE MANGER ARN>"
+	domain := "<YOUR DOMAIN>"
```

Comment:
Tests must run unattended, so either create these resources as part of the test (and clean them up afterward) or mock them.

Reply:
I rewrote the test code using mocks.